repose-aggregator/core/repose-core-api/src/main/resources/META-INF/schema/system-model/system-model.xsd

```diff
@@ -275,13 +283,153 @@
             </xs:annotation>
         </xs:attribute>
 
-        <xs:attribute name="uri-regex" type="xs:string" use="optional" default=".*">
+        <xs:attribute name="uri-regex" type="xs:string" use="optional">
             <xs:annotation>
                 <xs:documentation>
-                    <html:p>Regex of what uri is allowed access.</html:p>
+                    <html:p>
+                        Deprecated: This attribute is deprecated and will be removed in future major release.
+                        Regex of what uri is allowed access.
+                    </html:p>
                 </xs:documentation>
             </xs:annotation>
         </xs:attribute>
+        <xs:assert vc:minVersion="1.1"
+                   xpathDefaultNamespace="##targetNamespace"
+                   test="not(@uri-regex and uri)"
+                   xerces:message="Cannot define both a deprecated uri-regex attribute and the new uri element"
```

Comment:
We aren't going to allow uri regex with any of the new criteria.

Reply:
Ok. I'll update that.